src/main/java/org/galatea/starter/domain/IexHistoricalPriceEntity.java

```diff
@@ -0,0 +1,80 @@
+package org.galatea.starter.domain;
+
+import java.math.BigDecimal;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
+
+/**
+ * This class represents an entity of the IexHistoricalPrice object.  It will be used to store these
+ * objects in an h2 database.
+ */
+
+@Data
+@Builder
+@Entity
+@AllArgsConstructor(access = AccessLevel.PRIVATE)
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+public class IexHistoricalPriceEntity {
+
+  @Id
+  protected String symbol_date;
+
+  @NonNull
+  protected LocalDate date;
+
+  @NonNull
+  protected String symbol;
+
+  @NonNull
+  protected BigDecimal close;
+
+  @NonNull
+  protected BigDecimal high;
+
+  @NonNull
+  protected BigDecimal low;
+
+  @NonNull
+  protected BigDecimal open;
+
+  @NonNull
+  protected BigDecimal volume;
+
+  @NonNull
+  String dateString;
+
+
+  public static List<IexHistoricalPriceEntity> createFromHistoricalPriceList(
+      List<IexHistoricalPrice> historicalPrices) {
+    List<IexHistoricalPriceEntity> historicalPriceEntities =
+        new ArrayList<>();
+    String symbol_date;
```

Comment:
Java convention is `camelCase` for variables like this. Also you could just define these in your loop on line 63 since you never use these variables outside of that loop.